src/main/java/bio/terra/workspace/service/job/JobService.java

```diff
@@ -74,6 +74,7 @@ public JobService(
         Stairway.newBuilder()
             .applicationContext(applicationContext)
             .exceptionSerializer(serializer)
+            .keepFlightLog(true)
```

Comment:
to confirm: this means stairway will no longer delete flights from it's DB once they're finished?

Reply:
Stairway does not delete flights at all right now, but it does delete the flight logs. This stops it from deleting the flight logs. The plan is to have a retention policy set here (like 90d) and delete after that.